packages/library/src/commonMain/kotlin/io/realm/internal/SuspendableNotifier.kt

```diff
@@ -138,6 +139,27 @@ internal class SuspendableNotifier(private val owner: Realm, private val dispatc
         }
     }
 
+    /**
```

Comment:
This looks extremely like the other implementations. Can't we somehow generic-alize into `<T: Observable> changed(t: T): Flow<T> { ... }` and put the differentiation into the `Observables` (object, results, lists).